src/mumble/ServerLogs.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "ServerLogs.h"
+#include "MainWindow.h"
+#include "ServerHandler.h"
+#include "Version.h"
+#include "Global.h"
+
+#include <QTableWidgetItem>
+
+#include <boost/accumulators/accumulators.hpp>
```

Comment:
These seem to have been introduced by copy&pasting from somewhere else and can be removed (unless those you actually need)